sdk/data/azcosmos/ci.yml

```diff
@@ -26,16 +26,18 @@ stages:
     ServiceDirectory: 'data/azcosmos'
 - stage: Emulator
   displayName: 'Cosmos Emulator'
+  variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
   jobs:
   - job: DownloadAndRunCosmosEmulator
     displayName: Download and run Cosmos Emulator
 
     strategy:
       matrix:
-        Windows_Go118:
+        Windows_Go120:
           pool.name: azsdk-pool-mms-win-2022-general
           image.name: MMS2022
-          go.version: '1.18'
+          go.version: '1.20.7'
```

Comment:
You could use `1.21.1` which is the latest.

Reply:
Done